Task queue will detect this failure and this task will be retried later.

After update digdag 0.9.18, I encountered this error.

```
2018-04-24 16:53:30 +0900: Digdag v0.9.18
2018-04-24 16:53:31 +0900 [INFO] (main): Setting project path to /path/to/digdag/digdag-plugin-mysql/sample_local
2018-04-24 16:53:31 +0900 [WARN] (main): Reusing the last session time 2018-04-24T00:00:00+00:00.
2018-04-24 16:53:31 +0900 [INFO] (main): Using session /path/to/digdag/digdag-plugin-mysql/sample_local/.digdag/status/20180424T000000+0000.
2018-04-24 16:53:31 +0900 [INFO] (main): Starting a new session project id=1 workflow name=plugin session_time=2018-04-24T00:00:00+00:00
2018-04-24 16:53:32 +0900 [INFO] (0017@[0:default]+plugin+step1): mysql>: test.sql
2018-04-24 16:53:32 +0900 [ERROR] (task-thread-0): Uncaught exception. Task queue will detect this failure and this task will be retried later.
java.lang.IllegalAccessError: tried to access method io.digdag.standards.operator.jdbc.AbstractJdbcOperator.<init>(Lio/digdag/spi/OperatorContext;Lio/digdag/spi/TemplateEngine;)V from class com.github.hiroyuki_sato.digdag.plugin.mysql.MysqlOperatorFactory$MysqlOperator
	at com.github.hiroyuki_sato.digdag.plugin.mysql.MysqlOperatorFactory$MysqlOperator.<init>(MysqlOperatorFactory.java:44)
	at com.github.hiroyuki_sato.digdag.plugin.mysql.MysqlOperatorFactory.newOperator(MysqlOperatorFactory.java:36)
	at io.digdag.core.agent.OperatorManager.callExecutor(OperatorManager.java:310)
	at io.digdag.cli.Run$OperatorManagerWithSkip.callExecutor(Run.java:694)
	at io.digdag.core.agent.OperatorManager.runWithWorkspace(OperatorManager.java:254)
	at io.digdag.core.agent.OperatorManager.lambda$runWithHeartbeat$2(OperatorManager.java:137)
	at io.digdag.core.agent.LocalWorkspaceManager.withExtractedArchive(LocalWorkspaceManager.java:25)
	at io.digdag.core.agent.OperatorManager.runWithHeartbeat(OperatorManager.java:135)
	at io.digdag.core.agent.OperatorManager.run(OperatorManager.java:119)
	at io.digdag.cli.Run$OperatorManagerWithSkip.run(Run.java:676)
	at io.digdag.core.agent.MultiThreadAgent.lambda$null$0(MultiThreadAgent.java:127)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
^Chsato-mbp13:digdag-plugin-mysql hsato$ digdag selfupdate
```

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # digdag-plugin-mysql
 
-Digdag `mysql>` operator plugin to execute a query on MySQL sserver.
+Digdag `mysql>` operator plugin to execute a query on MySQL server.
 
 ## configuration
 
@@ -12,10 +12,10 @@ _export:
     repositories:
     repositories:
       #- file://${repos}
-      - file:///path/to/digdag-plugin-mysql/build/repo
-      # - https://jitpack.io
+      #- file:///path/to/digdag-plugin-mysql/build/repo
+      - https://jitpack.io
     dependencies:
-      - com.github.hiroyuki-sato:digdag-plugin-mysql:0.1.0
+      - com.github.hiroyuki-sato:digdag-plugin-mysql:0.1.1
 
   mysql:
     host: localhost

--- a/build.gradle
+++ b/build.gradle
@@ -13,7 +13,7 @@ apply plugin: 'net.ltgt.apt'
 group = 'com.github.hiroyuki-sato'
 version = '0.1.0'
 
-def digdagVersion = '0.9.16'
+def digdagVersion = '0.9.24'
 
 repositories {
     mavenCentral()

--- a/src/main/java/com/github/hiroyuki_sato/digdag/plugin/mysql/MysqlOperatorFactory.java
+++ b/src/main/java/com/github/hiroyuki_sato/digdag/plugin/mysql/MysqlOperatorFactory.java
@@ -16,12 +16,14 @@ public class MysqlOperatorFactory
     private static final String OPERATOR_TYPE = "mysql";
 
     private final TemplateEngine templateEngine;
+    private final Config systemConfig;
 
     private static Logger logger = LoggerFactory.getLogger(MysqlOperatorFactory.class);
 
-    public MysqlOperatorFactory(TemplateEngine templateEngine)
+    public MysqlOperatorFactory(Config systemConfig,TemplateEngine templateEngine)
     {
         this.templateEngine = templateEngine;
+        this.systemConfig = systemConfig;
     }
 
     @Override
@@ -33,15 +35,15 @@ public class MysqlOperatorFactory
     @Override
     public Operator newOperator(OperatorContext context)
     {
-        return new MysqlOperator(context,templateEngine);
+        return new MysqlOperator(systemConfig,context,templateEngine);
     }
 
     static class MysqlOperator
             extends AbstractJdbcJobOperator<MysqlConnectionConfig>
     {
-        MysqlOperator(OperatorContext context, TemplateEngine templateEngine)
+        MysqlOperator(Config systemConfig,OperatorContext context, TemplateEngine templateEngine)
         {
-            super(context, templateEngine);
+            super(systemConfig,context, templateEngine);
         }
 
         @Override

--- a/src/main/java/com/github/hiroyuki_sato/digdag/plugin/mysql/MysqlPlugin.java
+++ b/src/main/java/com/github/hiroyuki_sato/digdag/plugin/mysql/MysqlPlugin.java
@@ -1,5 +1,6 @@
 package com.github.hiroyuki_sato.digdag.plugin.mysql;
 
+import io.digdag.client.config.Config;
 import io.digdag.spi.OperatorFactory;
 import io.digdag.spi.OperatorProvider;
 import io.digdag.spi.Plugin;
@@ -24,9 +25,12 @@ public class MysqlPlugin implements Plugin {
         @Inject
         protected TemplateEngine templateEngine;
 
+        @Inject
+        protected Config systemConfig;
+
         @Override
         public List<OperatorFactory> get() {
-            return Arrays.asList(new MysqlOperatorFactory(templateEngine));
+            return Arrays.asList(new MysqlOperatorFactory(systemConfig,templateEngine));
         }
     }
 }

